pages/made-in-youngstown.tsx

```diff
@@ -1,17 +1,40 @@
 import Layout from "../components/Layout";
-import Companies from "../components/Companies";
+import Companies from "../components/MadeInYoungstown/Companies";
+import AddCompanyForm from "../components/MadeInYoungstown/AddCompanyForm";
+import { useState } from "react";
 
 const pageDescription =
   "Local tech companies in and around the Youngstown OH area.";
 
 export default function MadeInYoungstown() {
+  const [formVisibility, setFormVisibility] = useState("hidden");
+
+  const toggleFormModal = () => {
+    formVisibility == "hidden"
+      ? setFormVisibility("flex")
+      : setFormVisibility("hidden");
```

Comment:
Instead of using hidden, you should just not include the element in the dom next render, like

`{formVisibility && <AddCompanyForm toggleClose={toggleFormModal} />}`